website/static/js/pages/sharing-page.js

```diff
@@ -40,3 +88,48 @@ $(function() {
         trigger: 'hover'
     });
 });
+
+var checkWindowWidth = function() {
+    if ($(window).width() <= 600) {
+        $('table.responsive-table-xxs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
+    }
+    else {
+        $('table.responsive-table-xxs tbody tr:not(:hidden) td>div.header').css('display', '');
+        $('table.responsive-table-xxs tbody tr td:first-child').removeAttr('role').removeAttr('onclick');
+    }
+    if ($(window).width() <= 768) {
+        $('table.responsive-table-xs tbody tr td:first-child').attr('role','button').attr('onclick', 'toggleExpand(this.parentElement)');
```

Comment:
Why don't you use jquery for this onclick? It would allow you to scope your functions properly.